### PR TITLE
Add mXMR support

### PR DESCRIPTION
**Pull request by dginovker**
So people don't need to add 0.00... before every small tip.

### CODE REVIEW

File: tipperInteractions/comment_parse.py [hunk 1]
```diff
@@ -191,7 +187,7 @@ def parse_anontip_amount(self, subject):
         """
         :param subject: in format "Anonymous tip USER AMOUNT xmr"
         """
-        m = re.search('anonymous tip .+ (.+) xmr', subject.lower())
+        m = re.search('anonymous tip .+ (.+) (m)?xmr', subject.lower())
```
Comment (dginovker) [inline tipperInteractions/comment_parse.py]: Return needs to be adjusted

File: .gitignore [hunk 2]
```diff
@@ -4,6 +4,8 @@ tipperInteractions/__pycache__/
 wallets/
 praw.ini
 __pycache__/
+tests/_trial_temp/
 venv/
 logs/*
 monero/*
+tests/_trial_temp/
```
Comment (dginovker) [inline .gitignore]: Exists twice

**Comment by trasherdk** [comment]
How can there be a `below visible threshold` when you got: `mXMR` , `µXMR`, `nXMR` and `pXMR`?

**Comment by dginovker** [comment]
> How can there be a `below visible threshold` when you got: `mXMR` , `µXMR`, `nXMR` and `pXMR`?

It still lets the transaction go through, but by default the tip bot only displays 5 digits of your balance. Previously, if you had 0.00001 XMR, it would show up as "0 XMR". Now, it tells you that there's a small balance not being displayed.

**Pull request merged**